spotinst/provider.go

```diff
@@ -89,6 +89,12 @@ func Provider() *schema.Provider {
 
 			// Ocean Spark Virtual Node Group
 			string(commons.OceanSparkVirtualNodeGroupResourceName): resourceSpotinstOceanSparkVirtualNodeGroup(),
+
+			// AWS Account Creation
+			string(commons.AccountAWSResourceName): resourceSpotinstAccountAWS(),
+
+			// AWS set credential
+			string(commons.SetCredentialAWSResourceName): resourceSpotinstCredentialAWS(),
```

Comment:
CredentialsAWSResourceName and resourceSpotinstCredentialsAWS()

Reply:
updated to string(commons.CredentialsAWSResourceName): resourceSpotinstCredentialsAWS(),